tests/framework/helpers/HtmlTest.php

```diff
@@ -1390,6 +1397,15 @@ public function testError()
      */
     public function testCollectError()
     {
+        /**
+         * compact() issues an E_NOTICE level error if a given string refers to an unset variable
+         * @see http://php.net/manual/en/function.compact.php
+         */
+        if (version_compare(phpversion(), '7.3.0', '>=')) {
+            $this->markTestSkipped('Unable to be tested on PHP >= 7.3');
```

Comment:
This is wrong, don't skip the test, rewrite it to *not* use `compact()`.

Reply:
@bizley `compact()` usage was removed